arangograph/backups.md

```diff
@@ -14,6 +14,7 @@ To backup data in ArangoGraph for an ArangoDB installation, navigate to the
 
 There are two ways to create backups. Create periodic backups using a
 **Backup policy**, or create a backup manually.
+Both ways allow you to create backups in multiple regions as well.
```

Comment:
Please say that this feature is available for enterprise tier only.

Reply:
It is already mentioned here https://deploy-preview-1192--zealous-morse-14392b.netlify.app/docs/3.10/arangograph/backups.html#multi-region-backups

Reply:
but we can add it here and below as well to avoid confusion

Reply:
or make it a link then to the section where it's mentioned